packages/twenty-server/src/core/billing/controllers/product-price.controller.ts

```diff
@@ -0,0 +1,48 @@
+import { Controller, Get, Req, Res } from '@nestjs/common';
+
+import { Request, Response } from 'express';
+
+import {
+  AvailableProducts,
+  BillingService,
+  PriceData,
+} from 'src/core/billing/billing.service';
+import { StripeService } from 'src/core/billing/stripe.service';
+
+@Controller('stripe/product-prices/*')
+export class ProductPriceController {
+  constructor(
+    private readonly stripeService: StripeService,
+    private readonly billingService: BillingService,
+  ) {}
+
+  @Get()
+  async get(
+    @Req() request: Request,
+    @Res() res: Response<PriceData | { error: string }>,
+  ) {
+    const product = request.path.replace(
```

Comment:
Shouldn't we use nestjs params here? I'm not an expert but we should have a normalized way to extract the product from the url as a route parameter